### DISCUSSION TITLE
os error 10061

### DISCUSSION THREAD

**Discussion by 8-21com** [ordinals/ord#3201] [Q&A]
Hello,

I am installing a node for first time with ord. 

I've installed Bitcoin Core on a dedicated Windows machine, and installed latest release of ord to create a wallet...

When I try to use the command for ord wallet balance, I get the error below. Any help is appreciated.

```

because: error trying to connect: tcp connect error: No connection could be made because the target machine actively refused it. (os error 10061)
because: tcp connect error: No connection could be made because the target machine actively refused it. (os error 10061)
because: No connection could be made because the target machine actively refused it. (os error 10061)

```

**Comment by AirdropsCripto** [comment]
I am having the same issue, did you resolve it? If yes can you say how.

**Comment by dodgyone23** [comment]
with the new version you have to do the command `ord server` first to start it or if your cookie file is in an abnormal location `ord --cookie-file "<Cookie File Location>" server`
